services/terraformer/templates/gcp-lb.tpl

```diff
@@ -50,39 +51,45 @@ resource "google_compute_firewall" "firewall-{{ $region }}" {
 {{- end }}
 
 {{- range $i, $nodepool := .NodePools }}
-resource "google_compute_subnetwork" "{{$nodepool.Name}}-subnet" {
-  provider      = google.lb-nodepool-{{ $nodepool.Region }}
+resource "google_compute_subnetwork" "{{ $nodepool.Name }}_subnet" {
+  provider      = google.lb_nodepool_{{ $nodepool.Region }}
   name          = "{{ $nodepool.Name }}-{{ $clusterHash }}-subnet"
-  network       = google_compute_network.network-{{ $nodepool.Region }}.self_link
-  ip_cidr_range = "{{getCIDR "10.0.0.0/24" 2 $i}}"
+  network       = google_compute_network.network_{{ $nodepool.Region }}.self_link
+  ip_cidr_range = "{{ getCIDR "10.0.0.0/24" 2 $i }}"
+  description   = "Managed by Claudie"
 }
 
-resource "google_compute_instance" "{{$nodepool.Name}}" {
-  provider     = google.lb-nodepool-{{ $nodepool.Region }}
-  count        = {{ $nodepool.Count }}
-  zone         = "{{ $nodepool.Zone }}"
-  name         = "{{ $clusterName }}-{{ $clusterHash }}-{{$nodepool.Name}}-${count.index + 1}"
-  machine_type = "{{$nodepool.ServerType}}"
+resource "google_compute_instance" "{{ $nodepool.Name }}" {
+  provider                  = google.lb_nodepool_{{ $nodepool.Region }}
+  count                     = {{ $nodepool.Count }}
+  zone                      = "{{ $nodepool.Zone }}"
+  name                      = "{{ $clusterName }}-{{ $clusterHash }}-{{ $nodepool.Name }}-${count.index + 1}"
+  machine_type              = "{{ $nodepool.ServerType }}"
+  description               = "Managed by Claudie"
   allow_stopping_for_update = true
   boot_disk {
     initialize_params {
       size = "{{ $nodepool.DiskSize }}"
-      image = "{{$nodepool.Image}}"
+      image = "{{ $nodepool.Image }}"
     }
   }
   network_interface {
-    subnetwork = google_compute_subnetwork.{{$nodepool.Name}}-subnet.self_link
+    subnetwork = google_compute_subnetwork.{{ $nodepool.Name }}_subnet.self_link
     access_config {}
   }
   metadata = {
     ssh-keys = "root:${file("./public.pem")}"
   }
   metadata_startup_script = "echo 'PermitRootLogin without-password' >> /etc/ssh/sshd_config && echo 'PubkeyAuthentication yes' >> /etc/ssh/sshd_config && service sshd restart"
+
+  labels = {
+    environment = "claudie"
```

Comment:
"Managed by claudie" wouldn't work here ?

Reply:
The GCP labels have a strict requirements, like no capitals, no white spaces. We can do `managed-by-claudie` though.